apps/experiments/models.py

```diff
@@ -1184,6 +1191,11 @@
         """Returns the default experiment, or if there is none, the working experiment"""
         return self.experiment.get_working_version()
 
+    @property
+    def experiment_version_for_display(self):
+        version_number = self.get_experiment_version_number()
+        return "Default version" if version_number == "default" else f"v{version_number}"
+
```

Comment:
This could actually be a good suggestions @SmittieC, using '0' as the version number for the default version. This would make the URL path params always an integer and generally make it more consistent.

What do you think?

Reply:
Yes this is a good suggestion. I'll make an update

Reply:
I used 0 for to indicate the default version in [this](https://github.com/dimagi/open-chat-studio/pull/698/commits/644c46042a7b975ee7bac93aa0fa7bc256b6e554) commit (644c46042a7b975ee7bac93aa0fa7bc256b6e554). @coderabbitai please review this commit